vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -13234,6 +13234,25 @@ unit:HR-PER-FT2
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "hour per square foot" ;
 .
+unit:HR-PER-YR
```

Comment:
To be even more sophisticated, consider adding a numerator and denominator dimension vector (each with dimension of time) as described [here](https://github.com/qudt/qudt-public-repo/wiki/User-Guide-for-QUDT#6-using-quantity-kinds-with-dimensionless-quantities).